Projekty/Projekt2/Deregowski_Jamrozy_Janus/P2_KM2_Deregowski_Jamrozy_Janus.ipynb

```diff
@@ -0,0 +1,1000 @@
+{
```

Comment:
<p>dobrze, że pokazano jak zmienia się liczba kolumn po wykonaniu operacji (oraz zwrócono uwagę na kolejność).</p><p>Szkoda, że nie wyjaśniono w dwóch zdaniach na czym polegają te metody aby odbiorca z brakiem wiedzy z NLP, też mógł zrozumieć. Słuszna była uwaga innego zespołu, że lematyzacja aby działa najlepiej gdy ma informację o kontekście (czyli mamy do czynienia ze zdaniami a nie pojedynczymi słowami). Niemniej jednak z uwagi na to, że nie jest to kurs NLP uważam to za stosunkowo mały niuans.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>